packages/camyll/camyll.0.4.3/opam

```diff
@@ -0,0 +1,65 @@
+opam-version: "2.0"
+synopsis: "A static site generator"
+description: """
+Camyll is a static site generator.
+
+Features:
+
+- Conversion from Markdown to HTML
+- Syntax highlighting of any language via user-provided TextMate grammars
+- Post tagging
+- Processing of literate Agda"""
+maintainer: ["Alan Hu <alanh@ccs.neu.edu>"]
+authors: ["Alan Hu <alanh@ccs.neu.edu>"]
+license: "MIT"
+tags: ["blog" "web" "website"]
+homepage: "https://alan-j-hu.github.io/camyll"
+bug-reports: "https://github.com/alan-j-hu/camyll/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "angstrom" {>= "0.15" & < "0.16"}
+  "calendar" {>= "2.01" & < "4"}
+  "cmdliner" {>= "1.1" & < "2"}
+  "ezjsonm" {>= "1.3" & < "2"}
+  "httpaf" {>= "0.7.1" & < "0.8"}
+  "httpaf-lwt-unix" {>= "0.7.1" & < "0.8"}
+  "jingoo" {>= "1.4" & < "2"}
+  "markup" {>= "0.8" & < "2"}
+  "ocaml" {>= "4.14"}
+  "omd" {>= "2.0.0~"}
+  "otoml" {>= "0.9.3"}
+  "plist-xml" {>= "0.5"}
+  "re" {>= "1.9" & < "2"}
+  "slug" {>= "1.0" & < "2"}
+  "textmate-language" {>= "0.3.2" & < "0.4"}
+  "uri" {>= "4.2" & < "5"}
+  "yaml" {>= "3.1" & < "4"}
```

Comment:
are all the upper-bound constraints really necessary? We try to avoid that when it's not necessary in opam-repository